docs/modules/ROOT/pages/how-tos/testing/integration/adding.adoc

```diff
@@ -40,3 +40,5 @@ When the new build is finished:
 .  On the *Details* page, you can see if the test succeeded for that component. Navigate to the other tabs for more details. 
 
 
+NOTE: Now you know how to add integration test, in case you want to make a custom one, please continue to next section: xref:./creating.adoc[Creating a custom integration test]
```

Comment:
Looking at this page overall, I think it might make sense to add this right after line 20. Maybe something like

```
. Optional: If you do not already have a custom test that you want to add, we tell you how to xref:./creating.adoc[create a custom integration test] in the next section.
```

Reply:
Actually maybe it would be better if it is, a note like 
```
+
NOTE: If you do not already have a custom test that you want to add, we tell you how to xref:./creating.adoc[create a custom integration test] in the next section.
```
after line 19.